src/v/syschecks/syschecks.h

```diff
@@ -26,6 +29,7 @@ namespace syschecks {
 extern ss::logger checklog;
 
 static inline void initialize_intrinsics() {
+#if !defined __arm__ && !defined __aarch64__
```

Comment:
do we need `__arm__` which I think is for 32-bit machines. that seems like something we shouldn't even bother with?

Reply:
thanks, I removed `__arm__`